Sources/BraintreeCore/Public/BraintreeCore/BTAppContextSwitcher.h

```diff
@@ -59,21 +59,13 @@ NS_ASSUME_NONNULL_BEGIN
 */
 + (void)setReturnURLScheme:(NSString *)returnURLScheme;
 
-/**
- Handles a return from app context switch
-
- @param url The URL that was opened to return to your app
- @return `YES` if the app switch successfully handled the URL, or `NO` if the attempt to handle the URL failed.
-*/
-+ (BOOL)handleOpenURL:(NSURL *)url NS_SWIFT_NAME(handleOpenURL(_:));
-
```

Comment:
Thinking through this again - I actually am not sure we should delete this method. I initially was thinking through the lens of "if a merchant is **creating** an app using XCode 13 and targeting at the minimum iOS 13+, it will be auto-configured to use scenes. Removing the scene support is not encouraged."

However I forgot to think of this case "A merchant who **created** their app in a previous Xcode version (before scenes), and hasn't updated to add scene support. Example - a merchant started their app in Xcode 11, iOS 12+. Now they're updating their app to run on Xcode 13 and support iOS 13+." In doing this, Xcode doesn't enforce adding scene support. Here's an example of when we [updating our Demo app](https://github.com/braintree/braintree_ios/commit/e61f74f82ee84c47d7477ddd76b6c251f6af56c1) to include scene support.

TLDR: I don't think all merchants will necessarily be adding scene support to older projects. We should keep this method around but clarify that this one is meant for use if your app "Does not support scenes" and the other method is meant for if your app "does support scenes".

Apologies for the back and forth on this one, Sarah! Let me know if this thinking makes sense.